composer.json

```diff
@@ -5,26 +5,34 @@
   "license": "GPL-2.0",
   "require-dev": {
     "php": ">=7.0",
-    "wp-coding-standards/wpcs": "^1.0.0",
+    "wp-coding-standards/wpcs": "^1.1.0",
```

Comment:
This shouldn't be necessary I believe, shouldn't `^1.0.0` get every version until the next major?

Reply:
Yeah, my bad, `1.0.0` should be set here, I'll change this :+1: